src/js/edu-benefits/1990n/containers/ConfirmationPage.jsx

```diff
@@ -52,15 +36,11 @@ class ConfirmationPage extends React.Component {
           <i>Please print this page for your records.</i>
         </p>
         <div className="inset">
-          <h4>Education Benefit Claim <span className="additional">(Form 22-1990e)</span></h4>
+          <h4>Education Benefit Claim <span className="additional">(Form 22-1990n)</span></h4>
           <span>for {name.first} {name.middle} {name.last} {name.suffix}</span>
 
           <ul className="claim-list">
             <li>
-              <strong>Selected benefit</strong><br/>
-              {benefitsLabels[benefit]}
```

Comment:
You don't select an actual benefit on this form, just check a box saying pick the best one for me.